#!/usr/bin/env python

PACKAGE="simulator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
# for pitch and roll
gen.add("Kp", double_t, 0, "Proportional gain (pitch and roll)", 52.2, 0.0, 60.0)
gen.add("Ki", double_t, 0, "Integral gain (pitch and roll)", 0.0, 0.0, 60.0)
gen.add("Kd", double_t, 0, "Derivative gain (pitch and roll)", 55.2, 0.0, 60.0)

# for yaw
gen.add("KpYaw", double_t, 0, "Proportional gain (yaw)", 5., 0.0, 60.0)
gen.add("KiYaw", double_t, 0, "Integral gain (yaw)", 0.0, 0.0, 60.0)
gen.add("KdYaw", double_t, 0, "Derivative gain (yaw)", 8., 0.0, 60.0)


exit(gen.generate(PACKAGE, "simulator","attitudeCtrlCFG"))
